.github/workflows/ci.yml

```diff
@@ -109,7 +109,7 @@ jobs:
           for f in ./test-binaries/*; do
             echo "running $f"
             chmod +x $f  # GH action artifacts don't handle permissions
-            $f --ignored || exit 1
+            $f --ignored --test-threads=1 || exit 1
```

Comment:
Hmm, why did you need to add this? (Maybe you have it in a commit message somewhere, but there are 47 commits so I haven't read them all individually :sweat_smile:)

Reply:
my commit messages are not really useful for these small increments, sorry about that. 

Our setup doesn't support running builds in parallel. After adding the second build test it failed because of that. 

I added a comment